README.md

```diff
@@ -198,6 +198,8 @@ Please note that development is only possible on Linux and MacOS, no Windows sup
 ### Build
 
 ```sh
+git clone https://github.com/home-assistant-libs/python-matter-server.git
```

Comment:
This is only needed the first time you start developing. After that you need another command to keep the local repository synced with the remote repository.

I don't think we should add these commands as they are common for all development for git repositories.